AppCenterCrashes/AppCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -1203,6 +1205,49 @@ - (void)testCrashesSetCorrectUserIdToLogs {
   [settings stopMocking];
 }
 
+#if !TARGET_OS_OSX
+
+- (void)testObserverAddedNotOsxNotExtension {
+
+  // When
+  [self.sut applyEnabledState:YES];
+
+  // Then
+  OCMVerify([MS_NOTIFICATION_CENTER addObserver:self.sut
```

Comment:
Maybe we should check that it actually responds to selctor?

Reply:
Well that was initial implementation 🙄